### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by ashutosh007865**
Enhance the readme.MD file for better understanding of project

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -3,6 +3,16 @@
 Not putting much here right now since this is a WIP. If you have feature requests, open up a new issue. I'll organize the code nicely soon enough.
 
 For now the only proper feature in it is the dice roll.
+## What is bot?
```
Comment (akashsara) [inline readme.md:6]: This section isn't needed.

File: readme.md [hunk 2]
```diff
@@ -3,6 +3,16 @@
...
+ Bot is a software application that runs automated tasks (scripts) over the Internet. Typically, bots perform tasks that are simple and repetitive, much faster than a person could. The most extensive use of bots is for web crawling, in which an automated script fetches, analyzes and files information from web servers. More than half of all web traffic is generated by bots.
+ ## Status of this Project
```
Comment (akashsara) [inline readme.md:8]: It can actually do more than that, so this will need to be updated.

File: readme.md [hunk 3]
```diff
@@ -3,6 +3,16 @@
...
+ Bot can do lots of things, but as now this project contains the fuctionality of roll dice only.
+ 
+ ## Contribution
```
Comment (akashsara) [inline readme.md:11]: This should be spell-checked and then moved into a contributions.md file

**Review by akashsara — CHANGES_REQUESTED** [review]

**Comment by ashutosh007865** [comment]
okay I will review the readme file again

**Pull request closed without merging**